internal/fs/file_windows.go

```diff
@@ -134,3 +136,20 @@ func openHandleForEA(nodeType restic.NodeType, path string, writeAccess bool) (h
 	}
 	return handle, err
 }
+
+// RecallOnDataAccess checks if a file is available locally on the disk or if the file is
+// just a placeholder which must be downloaded from a remote server. This is typically used
+// in cloud syncing services (e.g. OneDrive) to prevent downloading files from cloud storage
+// until they are accessed.
+func RecallOnDataAccess(fi *ExtendedFileInfo) (bool, error) {
```

Comment:
Please move that function to the `stat_*.go` files and change its signature to `func (fi *ExtendedFileInfo) RecallOnDataAccess() (bool, error)`.

Reply:
(This will require duplicating the RecallOnDataAccess placeholder for both `stat_unix` and `stat_bsd`)